keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -417,14 +414,18 @@ affect the perceived performance.
 3. As releases increment, the probablity of a WebSocket enabled `kubectl` communicating
 with an older non-WebSocket enabled API Server decreases.
 
-### Proposal: `K8s-Websocket-Protocol: stream-translate` Header
+### Proposal: New `RemoteCommand` Sub-Protocol Version - `v5.channel.k8s.io`
 
-In addition to the current SPDY-based clients, there are other current WebSocket clients,
-including a javascript/browser-based client. In order to distinguish these older
-WebSocket clients from the new stream-translated WebSocket clients, we propose adding
-a new header `K8s-Websocket-Protocol: stream-translate`. As described further in the
-next `Proposal` section, this header allows newer clients to delegate to the
-`StreamTranslatorProxy` to translate WebSockets data messages to SPDY.
+The latest RemoteCommand version does not address an important protocol feature--a
+stream `CLOSE` signal. In order to communicate to another endpoint that the current
+stream of sending data is complete, a `CLOSE` signal is necessary. This problem
+currently arises when sending data over the STDIN stream, and it is more fully described
+in the following issue: [exec over web sockets protocol is flawed](https://github.com/kubernetes/kubernetes/issues/89899).
+A new RemoteCommand version (`v5.channel.k8s.io`) adds this `CLOSE` signal.
+
+### Proposal: New `PortForward` Sub-Protocol Version - `v2.portforward.k8s.io`
```

Comment:
either drop this section or clarify how resolution of this fits into the alpha timeline (required before alpha, required during alpha, etc)

Reply:
Dropped.